content/en/real_user_monitoring/product_analytics/sankey.md

```diff
@@ -17,7 +17,7 @@ Each node represents a view the user visited. The thickness of each node represe
 
 If a user visits the same page multiple times during their session, that page is only counted once.
 
-Actions are not supported in Sankey diagrams.
+Actions are not currently supported in the Sankey diagram.
```

Comment:
```suggestion
Actions are not supported in the Sankey diagram.
```
Our style recommends avoiding words like "currently" so it's clear our docs always reflect the current state of the product.